server/server.py

```diff
@@ -54,8 +54,8 @@ def tokenCount(self, content: Union[str, Dict[str, str],  List[Dict[str, str]]])
 MODELS: Dict[str, OpenAiModel] = {
     GPT3: OpenAiModel(GPT3, "GPT3💲", 0.002 / 1000, 0.0015 / 1000, 1024 * 4 - 1, tiktoken.encoding_for_model(GPT3)),
     GPT4: OpenAiModel(GPT4, "GPT4💵", 0.06 / 1000, 0.03 / 1000, 1024 * 8 - 1, tiktoken.encoding_for_model(GPT4)),
-    GPT4_32K: OpenAiModel(GPT4_32K, "GPT4-32K💰", 0.012 / 1000, 0.06 /
-                          1000, 1024 * 32 - 1, tiktoken.encoding_for_model(GPT4_32K))
+    GPT4_TURBO: OpenAiModel(GPT4_TURBO, "GPT4 Turbo💵", 0.01 / 1000, 0.03 /
+                          1000, 128000 - 1, tiktoken.encoding_for_model(GPT4_TURBO))
```

Comment:
I'm a little suspicious of this number, but it's what I found in the same place as earlier models with their [binary precise-numbers](https://platform.openai.com/docs/models/gpt-4-and-gpt-4-turbo).

Reply:
According to the annoucement:
> We also optimized its performance so we are able to offer GPT-4 Turbo at a [3x cheaper](https://openai.com/pricing#gpt-4-turbo) price for input tokens and a 2x cheaper price for output tokens compared to GPT-4

So probably 
```python
OpenAiModel(GPT4_TURBO, "GPT4 Turbo", 0.03 / 1000, 0.01 / 1000, 128 * 1000 - 1, tiktoken.encoding_for_model(GPT4_TURBO))
```
Since the comepletion tokens (expensive part) is first in the dataclass.
Wow thats gonna be some savings.   Gonna be some work to make a nice UI around the image handling and assistant API.  Funny that they call the image API a dev preview but its already part of ChatGPT.